sdk/communication/communication-calling-server/src/callConnection.ts

```diff
@@ -58,10 +58,10 @@ export interface CallConnection {
   /**
    * Play audio.
    *
-   * @param audioFileUri - The id for the media in the AudioFileUri, using which we cache the media resource.
+   * @param audioUrl - The audio resource url.
```

Comment:
wondering if we should call it audioResourceUrl

Reply:
use audioUrl per discussion